### ISSUE TITLE
Add DAC_Config_Creator

### ISSUE THREAD

**Issue by Przemro** [SavageCDN/DAC_A3#13]
Coud you add DAC_Config_Creator to this repo? I gues everyone use diffrent options, yet there is one thing that should be updated with units in DAC_Config_Creator.

`DAC_SP_Soldiers     =   ["B_soldier_AR_F","B_G_soldier_AR_F","O_soldier_AR_F","O_soldierU_AR_F","O_G_soldier_AR_F","I_soldier_AR_F","I_G_soldier_AR_F"];`

There should be added every AR/MG unit so DAC use them to do suppresive fire. Or maybe any other idea how could this be added so players can use it.

**Comment by SavageCDN** [comment]
Good idea - should ALL MG/AR units from all mods be put here?  Or rely on mission maker adding the right classnames for suppressive fire?

**Comment by Przemro** [comment]
Well I think that this is their job - to suppers, so all should be added. It will save time for others mission makers also with finding AR/MG, IMO it is easier to remove than search and add.

**Comment by SavageCDN** [comment]
Will get this updated later in the week sorry for delay

**Comment by SavageCDN** [comment]
Ooops I'm getting old :)  Will add this in a day or so.
